### ISSUE TITLE
Can monitoring MPM running on multiple machines simultaneously be done?

### ISSUE THREAD

**Issue by spotopolis** [MultiPoolMiner/MultiPoolMiner#2196]
I like the new local web monitor that MPM runs, is there a way to view the info of multiple machines running MPM on my network on a single page? I know there is the MPM status page from the website, but I would like to be able to view all my machines in a single page built from the MPM api.

**Comment by UselessGuru** [comment]
Sorry mate, this is currently not possible.

**Comment by paulpoco** [comment]
You would have to have a key for machine?  And then save a bookmark with that key for each machine?  It wouldn't be on the same page all together.

**Comment by giannibonfanti** [comment]
Hi @spotopolis ,
if you are interested in try something different than the monitoring page on the website, I can suggest you to try our project www.brucomining.com that we just started. You can have a private dashboard of all your machines, notifications and so on. More things will come.

It would be also wonderful if this could be natively integrated to MPM, maybe as an option to classic MPM monitoring server. @UselessGuru what do you think?

**Comment by grantemsley** [comment]
@giannibonfanti your fork removes us from the donations and puts yourself in.  I mean, it's open source software, and so you're free to do whatever you want as long as you disclose the source code, but that isn't the best way to say thank you for all the hard work we've done that you're basing your code on.

Looks like all you've really done is hard code the status reporting URL to your own.

The screenshots of your web interface look pretty nice.  Did you make that yourself or is it based on mpm-monitor?

**Comment by m-bosch** [comment]
@grantemsley @giannibonfanti the link to the miner is changed to the original of MPM:

https://www.brucomining.com/download

**Comment by giannibonfanti** [comment]
@m-bosch yes exactly, you can use BrucoMining monitoring with original MPM.
You just need to get a key and set MinerStatus* parameters as per instruction.
If you need assistance, just write me here or on our support page.

**Comment by jake5253** [comment]
why not use https://multipoolminer.io/monitor ?
